src/components/UI/Footer/FooterNav.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const FooterNav = styled.nav`
+	background-color: #342293;
+	width: 100%;
+	bottom: 0;
+	right: 0;
+	left: 0;
+	position: fixed;
```

Comment:
I think with a width of 100% you don't need left/right:0

Reply:
You are right, I removed it!